stix2/sources/__init__.py

```diff
@@ -28,14 +24,100 @@ def make_id():
     return str(uuid.uuid4())
 
 
-class DataStore(object):
+def apply_common_filters(stix_objs, query):
```

Comment:
Does this function now make more sense in the filters.py file?

Reply:
Yes